nodes/DelName.py

```diff
@@ -1 +1,12 @@
+"""
+DelName astroid node
+
+Attributes:
+    - targets  (List[Node])
```

Comment:
There might be the same kind of confusion here as with Decorators vs. FunctionDef.

It seems like you're saying both `Delete` and `DelName` nodes have an attribute called `target`, but this seems unlikely. (My intuition would be that `Delete` has the `target` attribute, which is a list of `DelName` and/or `DelAttr` nodes. I could be wrong, though.)